max_execution_time_request.awk .awk

```diff
@@ -0,0 +1,14 @@
+awk '{
+    for (i = 1; i <= NF; i++) {
+        if ($i == "in") {
+            time = $(i + 1); 
+            gsub(/ms/, "", time); 
+            records[NR] = $0; times[NR] = time
+        }
+    }
+}
+END {
```

Comment:
can you try to create a single line command for getting the max request time

Reply:
Okay let me try and update it

Reply:
@burhanuddin7 Done. I had tried 1 line command for getting complete request also and added it its file.
Note: i have also kept the previous commands for both scripts